examples/esp-idf-component/README.md

```diff
@@ -1,16 +1,17 @@
 # Using Micro-RDK as a library Example
 
-This example cover shows how to use Micro-RDK into a new or existing ESP project.
+This example cover shows how to use the Micro-RDK into a new or existing ESP project.
```

Comment:
This sentence looks a little off: "cover shows", "use ... into".

Reply:
maybe "for a new or existing..."?

Reply:
```suggestion
This example shows how to use the Micro-RDK in a new or existing ESP project.
```